sdk/rust/src/llm.rs

```diff
@@ -7,19 +7,21 @@ pub use crate::wit::fermyon::spin::llm::{
 
 /// The model use for inferencing
 #[allow(missing_docs)]
+#[derive(Debug, Clone, Copy)]
 pub enum InferencingModel<'a> {
     Llama2Chat,
     CodellamaInstruct,
     Other(&'a str),
 }
 
 impl<'a> InferencingModel<'a> {
-    fn as_str(&self) -> llm::InferencingModel<'a> {
+    fn to_string(&self) -> llm::InferencingModel {
```

Comment:
Not sure why this change is needed?

Reply:
It's not need but the type changed and `as_str` seemed wrong given it was returning a String and not a `str`. I've since changes this to a `Display` impl.